### PR TITLE
Release azure-cosmos 4.63.4-hotfix with version bump.

### PR DESCRIPTION
**Pull request by jeet1995**

### CODE REVIEW

**Comment by jeet1995** [comment]
CI run: https://dev.azure.com/azure-sdk/public/_build/results?buildId=4227936&view=results
Live test run: https://dev.azure.com/azure-sdk/internal/_build/results?buildId=4227328&view=results

File: eng/versioning/version_client.txt [hunk 1]
```diff
@@ -101,7 +101,7 @@ com.azure:azure-core-test;1.26.2;1.27.0-beta.1
 com.azure:azure-core-tracing-opentelemetry;1.0.0-beta.48;1.0.0-beta.49
 com.azure:azure-core-tracing-opentelemetry-samples;1.0.0-beta.1;1.0.0-beta.1
 com.azure:azure-core-version-tests;1.0.0-beta.1;1.0.0-beta.1
-com.azure:azure-cosmos;4.63.2;4.63.3
+com.azure:azure-cosmos;4.63.2;4.63.3-hotfix
```
Comment (tvaron3) [inline eng/versioning/version_client.txt:104]: Shouldn't the version be 4.63.4? I saw in the changelog the version still increased with hotfixes.

Comment (jeet1995) [reply eng/versioning/version_client.txt]: You're right - fixed it in next iteration.

**Review by tvaron3 — APPROVED** [review]
LGTM

**Pull request closed without merging**